tools/android_custom_build/build_custom_android_package.py

```diff
@@ -26,6 +26,16 @@ def parse_args():
         "working_dir", type=pathlib.Path, help="The directory used to store intermediate and output files."
     )
 
+    default_docker_psuedo_tty = True
+    parser.add_argument(
+        "--docker_psuedo_tty",
+        default=default_docker_psuedo_tty,
```

Comment:
consider something like `--disable_docker_pseudo_tty` with `action="store_false"`
https://docs.python.org/3/library/argparse.html#action

Reply:
I've addressed this in the commit [92b0197](https://github.com/microsoft/onnxruntime/pull/13704/commits/92b019772a0c1b90c614eb3124df8a8290db7cad), meanwhile, as we have decided to go with removing `-it` flag directly, this would not be part of the change, thus resolving this.